packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -55,6 +55,23 @@ export class FASTAnchor extends Anchor {
             this.appearance = "neutral";
         }
     }
+
+    /**
+     * Applies 'icon-only' class when there is only an SVG in the default slot
+     *
+     * @public
+     * @remarks
+     */
+    public defaultSlottedContentChanged(oldValue, newValue): void {
```

Comment:
I would recommend looking into using the `filter` option on the slotted directive. It works the same as for the `children` directive. Here's a link to the docs: https://www.fast.design/docs/fast-element/using-directives#the-children-directive

Reply:
@EisenbergEffect - I may misunderstand it, but I thought filter would only return those slotted elements. We want to support everything as slotted content here, but we only want to take action when it's an SVG and there is one item within the anchor.

Reply:
@EisenbergEffect @chrisdholt Since this is specifically for the use case of stand-alone SVG icons, would adding a filter for SVG elements in the template be too specific in FAST Foundation?

Reply:
Yep, I think this is a good callout - I'd prefer the generic (non-filtered) for foundation.

Reply:
Good point!